src/utils/formatting.py

```
@@ -0,0 +1,258 @@
+        new_turn: The message content
+        current_time: Timestamp string or "unknown"
+        speaker: The speaker's name
+
+    Returns:
+        Formatted string like "2023-05-08 13:56:00 speaker: hello" or "speaker: hello"
+    """
+    if isinstance(current_time, datetime):
+        current_time = current_time.strftime("%Y-%m-%d %H:%M:%S")
+    if current_time and current_time != "unknown":
+        return f"{current_time} {speaker}: {new_turn}"
+    else:
```

Ruff (0.12.2)

Unnecessary `else` after `return` statement

Remove unnecessary `else`

(RET505)